### DISCUSSION TITLE
Is there any way can delete the operators on sliver-server ?

### DISCUSSION THREAD

**Discussion by jieliau** [BishopFox/sliver#1790] [General]
Hi folks, 
Thanks for this awesome C2 framework. Just wonder if there is any way to delete one of the operators on Sliver-server. For example: 
[server] sliver > operators 

 Name   Status  
====== =========
 test   Offline 
 red    Offline 
 
 How do I delete test one?

**Comment by Arszilla** [comment]
Curious how to do this without nuking/pruning sliver as well - Did you manage to find an answer @jieliau?

**Reply by moloch--** [reply]
from the server console you can use `kick-operator`

**Reply by Researcher-maker** [reply]
Doesn't work for me...
[server] sliver > kick-operator

[*] Removing auth token(s) for , please wait ... 

[server] sliver > operators

 Name      Status  
========= =========
test   Offline

**Reply by mahatah** [reply]
The simple answer is `kick-operator -n <name_of_user_to_kick>`.

I believe the intended method to figure out to use the `-n` flag there is by using `kick-operator --help`, which displays the following information:
```
[server] sliver > kick-operator --help

Kick an operator from the server

Usage:
======
  kick-operator [flags]

Flags:
======
  -h, --help           display help
  -n, --name string    operator name
  ```
While it is more user friendly to remind the user that the `-n` flag, followed by the name of the operator to kick, is required when they run the `kick-operator` command using no flags, relying on the user to figure it out by supplying the `--help` flag is a reasonable expectation.

**Comment by PYfffE** [comment]
How can i kick operator if i use sliver-server as daemon?

**Reply by moloch--** [reply]
Start the `./sliver-server` (should be fine while the daemon is running) and run `kick-operator` though you bring up a good point I should probably add a `./sliver-server operator kick` command
